assets/composer/settings.local.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+if (getenv('OPENID_CONNECT_PARAMS') == true) {
+  $openid_config_params = json_decode(getenv('OPENID_CONNECT_PARAMS'), true);
```

Comment:
I don't think we need to manage any of these in the environment variables, but doesn't do any harm to have them.

Reply:
That's a hybrid blob of data for for the openid_connect module and x number of clients. One var per env. And the reason I've stored the openid_connect module's variables - there is one which we are certain to vary between envs: the placement of the buttons - above, or replacing the login form.